Riot/Modules/Common/Recents/RecentsViewController.m

```diff
@@ -1796,6 +1798,21 @@ - (void)onPlusButtonPressed
                                                        
                                                    }]];
     
+    [currentAlert addAction:[UIAlertAction
```

Comment:
We need to check if HS supports PSTN to add dial pad option. You can check it by `self.mainSession.callManager.supportsPSTN`

Reply:
Fixed in additional commit